src/loaders/gtdb/gtdb_genome_stats_loader.py

```diff
@@ -0,0 +1,131 @@
+     gtdb_genome_stats_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb
+     gtdb_genome_stats_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb --output  gtdb_genome_stats.json
+"""
+
+EXIST_FEATURES = {'accession', 'checkm_completeness', 'ncbi_contig_n50'
+                  }  # genome statistics already existing in the metadata files
+NON_EXIST_FEATURES = {'high_checkm_marker_count'}  # genome statistics to be computed
+HELPER_FEATURES = {'checkm_marker_count'}  # additional features needed to compute statistics
+
+# change header to specific context (default mapper: capitalize first character)
+HEADER_MAPPER = {'accession': 'Genome Name',
+                 'checkm_completeness': 'Completeness'}
+
+
+def _compute_stats(df, computations):
+    for computation in computations:
+        try:
+            comp_ops = getattr(helper, computation)
+        except KeyError as e:
+            raise ValueError(f'Please implement method [{computation}]') from e
+        result = comp_ops(df)
+        df[result.name] = result
+
+    return df
+
+
+def _parse_from_metadata_file(load_files, exist_features, additional_features):
+    frames = [pd.read_csv(load_file, sep='\t', header=0, usecols=exist_features.union(additional_features)) for
+              load_file in load_files]
+    df = pd.concat(frames, ignore_index=True)
+
+    return df
+
+
+def _rename_col(df, header_mapper):
+    mapper = {col: ' '.join(list(map(lambda x: x.capitalize(), col.split('_')))) for col in df.columns}
+    mapper.update(header_mapper)
+    df.rename(columns=mapper, errors="raise", inplace=True)
+
+
+def _row_to_doc(row, kbase_collection, load_version):
+    doc = {
+        "_key": hashlib.md5(
+            f"{kbase_collection}_{load_version}_{row.name}".encode('utf-8')
+        ).hexdigest(),
+        "Collection": kbase_collection,
+        "Load Version": load_version,
+    }
+    doc.update(row.to_dict())
+
+    return doc
+
+
+def _df_to_docs(df, kbase_collection, load_version, header_mapper):
+    _rename_col(df, header_mapper)
+    docs = df.apply(_row_to_doc, args=(kbase_collection, load_version), axis=1).to_list()
+
+    return docs
+
+
+def main():
+    if not all([header in EXIST_FEATURES.union(NON_EXIST_FEATURES) for header in HEADER_MAPPER.keys()]):
+        raise ValueError('Please make sure HEADER_MAPPER keys are all included in the FEATURES')
+
+    parser = argparse.ArgumentParser(
+        description='PROTOTYPE - Prepare GTDB genome statistics data in JSON format for arango import.')
+
+    # Required positional argument
+    parser.add_argument('load_files', type=argparse.FileType('r'), nargs='+',
+                        help='GTDB genome metadata files')
+
+    # Required flag argument
+    parser.add_argument(
+        '--load_version', required=True, type=str, nargs=1, help='KBase load version')
```

Comment:
I noticed this too. But we need this somehow. It will otherwise parse only the first character ('d') instead of ('dev-200') for example. I can dig more. But for now, `nargs=1` makes this script work.

Reply:
that's weird, it seems to work fine in my hands:
```
In [1]: import argparse

In [2]: p = argparse.ArgumentParser()

In [3]: p.add_argument("--foo", type=str, required=True, help="bar")
Out[3]: _StoreAction(option_strings=['--foo'], dest='foo', nargs=None, const=None, default=None, type=<class 'str'>, choices=None, required=True, help='bar', metavar=None)

In [4]: p.parse_args("--foo stuff".split())
Out[4]: Namespace(foo='stuff')
```

Reply:
Anyway, sounds like you want to fix this later, which is fine